Detections/MultipleDataSources/SecurityServiceRegistryACLModification.yaml

```diff
@@ -0,0 +1,86 @@
+
+id: 473d57e6-f787-435c-a16b-b38b51fa9a4b 
+name: Security Service Registry ACL Modification
+description: |
+    'Identifies attempts to modify registry ACL to evade security solutions. In the Solorigate attack, the attackers were found modifying registry permissions so services.exe cannot access the relevant registry keys to start the service.
+     The detection leverages Security Event as well as MDE data to identify when specific security services registry permissions are modified. 
+     Only some portions of this detection are related to Solorigate, it also includes coverage for some common tools that perform this activity. 
+     Reference on guidance for enabling registry auditing:
+     - https://docs.microsoft.com/windows/security/threat-protection/auditing/advanced-security-auditing-faq
+     - https://docs.microsoft.com/windows/security/threat-protection/auditing/appendix-a-security-monitoring-recommendations-for-many-audit-events
+     - https://docs.microsoft.com/windows/security/threat-protection/auditing/audit-registry
+     - https://docs.microsoft.com/windows/security/threat-protection/auditing/event-4670
+     - https://github.com/OTRF/Set-AuditRule 
+     - https://docs.microsoft.com/en-us/dotnet/api/system.security.accesscontrol.registryrights?view=dotnet-plat-ext-5.0
+     For the above event to be created the audit policy for the registry must have auditing enabled for Write DAC and/or Write Owner'
```

Comment:
This should go with the event-4670 line not under all the references.

Reply:
Done now. Thanks